src/local/butler/weights.py

```diff
@@ -137,21 +202,17 @@ def _fuzzer_job_to_dict(
   }
 
 
-def _dump_fuzzer_jobs() -> None:
+def _dump_fuzzer_jobs(fuzzer_jobs: Sequence[data_types.FuzzerJob]) -> None:
   """Dumps FuzzerJob entries from the database to stdout in CSV format."""
```

Comment:
Similarly to the list* counterparts, modify the description to 
'Dumps the provided FuzzerJob entries to stdout in CSV format.'

Reply:
Done.